pkg/agent/manager/manager.go

```diff
@@ -23,7 +23,7 @@ type Manager interface {
 
 	// Subscribe returns a channel on which cache entry updates are sent
 	// for a particular set of selectors.
-	Subscribe(key cache.Selectors, done chan struct{}) chan []cache.Entry
+	Subscribe(key cache.Selectors, done chan struct{}) chan *cache.Entry
```

Comment:
I think this should be a slice? The workload API response must return the "full" set of information in every update... that probably has some implications on exactly how subscribers is hooked into the manager. You might find it useful to read through the Workload API spec to better understand the behavior we'll need to implement there